app/Providers/AppServiceProvider.php

```diff
@@ -17,6 +17,7 @@ public function register(): void
             $this->app->register(\Laravel\Telescope\TelescopeServiceProvider::class);
             $this->app->register(TelescopeServiceProvider::class);
         }
+        $this->loadViewsFrom(base_path('app/Website/resources/views'), 'website');
```

Comment:
FYI there is a app_path, so you can make it a bit shorter: `app_path('Website/resources/views')`